slick.grid.js

```diff
@@ -913,6 +913,91 @@ if (typeof Slick === "undefined") {
       }
     }
 
+    function sortColumnByActiveCell() {
+      var $col;
+      var activeCell;
+      activeCell = getActiveCell();
+      if (activeCell) {
```

Comment:
let's invert the logic and return if the execution shouldn't continue to reduce the nested if blocks.

Reply:
same applies to the code below.